packages/dids/src/did.methods/did.key.manager.ts

```diff
@@ -0,0 +1,48 @@
+import { driver } from 'did-method-key';
+import { IDidManager } from '../interface/i.did.manager';
+import {Wallet} from '@docknetwork/wallet-sdk-core/lib/modules/wallet';
+
+export class DidKeyManager  implements IDidManager{
+  private static instance: IDidManager;
+
+
+  private wallet;
+
+  private constructor() {
+    this.wallet = Wallet.getInstance();
+  }
+  async createDID(options = {}): Promise<string> {
```

Comment:
creating a did string isnt really very useful, in this case itd be better to return the whole document